src/hardware/openbench-logic-sniffer/api.c

```diff
@@ -36,7 +36,7 @@ static const uint32_t devopts[] = {
 	SR_CONF_SAMPLERATE | SR_CONF_GET | SR_CONF_SET | SR_CONF_LIST,
 	SR_CONF_TRIGGER_MATCH | SR_CONF_LIST,
 	SR_CONF_CAPTURE_RATIO | SR_CONF_GET | SR_CONF_SET,
-	SR_CONF_EXTERNAL_CLOCK | SR_CONF_SET,
+	SR_CONF_EXTERNAL_CLOCK | SR_CONF_GET | SR_CONF_SET,
```

Comment:
Looks good again. Maybe should be closer to the patch "Add option to use the falling edge of the external clock"? Dunno if it conflicts with the huge trigger setup refactor patch. If so, just leave it here

Reply:
Luckily, it doesn't. Moved it immediately after the aforementioned commit.